next/components/forms/segments/AccountSections/TaxesFeesSection/TaxesFeesTaxAdministratorCard.tsx

```diff
@@ -1,20 +1,37 @@
 import { ClockIcon, MailIcon, PhoneIcon } from '@assets/ui-icons'
-import { StrapiTaxAdministrator } from '@backend/utils/tax-administrator'
+import { StrapiTaxAdministrator } from '@backend/utils/strapi-tax-administrator'
+import { ResponseTaxAdministratorDto } from 'openapi-clients/tax'
 
 import MLinkNew from '../../../simple-components/MLinkNew'
 
 type TaxesFeesTaxAdministratorCardProps = {
-  taxAdministrator: StrapiTaxAdministrator
+  beTaxAdministrator: ResponseTaxAdministratorDto | null
+  strapiTaxAdministrator: StrapiTaxAdministrator | null
+}
+
+const normalizeBeTaxAdministrator = (taxAdministrator: ResponseTaxAdministratorDto) => {
+  return {
+    name: taxAdministrator.name,
+    phone: taxAdministrator.phoneNumber,
+    email: taxAdministrator.email,
+  }
 }
 
 /**
  * TODO: Use card component, translations
- * @param taxAdministrator
- * @constructor
  */
 const TaxesFeesTaxAdministratorCard = ({
-  taxAdministrator,
+  beTaxAdministrator,
+  strapiTaxAdministrator,
 }: TaxesFeesTaxAdministratorCardProps) => {
+  if (!beTaxAdministrator && !strapiTaxAdministrator) {
+    return null
+  }
+
+  const taxAdministrator = beTaxAdministrator
+    ? normalizeBeTaxAdministrator(beTaxAdministrator)
+    : strapiTaxAdministrator!
```

Comment:
`strapiTaxAdministrator` cas still be null in this case, using `!` is not correct